### ISSUE TITLE
Data with ANSI colors in it breaks the column widths

### ISSUE THREAD

**Issue by scottchiefbaker** [neilb/Text-Table-Tiny#4]
If data contains ANSI color sequences, the column widths are calculated incorrectly. I think this could be fixed easily by adding add an option in `generate_table()` lwhich accounts for ANSI colors, and then using `colorstrip()` from `Term::ANSIColor` to calculate the lengths.

Simplified use case:

```
use Term::ANSIColor;

# Code here
my $red   = color("bold red");
my $reset = color("reset");

use Text::Table::Tiny 0.04 qw/ generate_table /;

my $rows = [
    # header row
    ['Name', 'Rank', 'Serial'],
    # rows
    ['alice', 'pvt', '123456'],
    ['bob',   $red . "cpl" . $reset, '98765321'],
    ['carol', 'brig gen', '8745'],
];
print generate_table(rows => $rows, header_row => 1) . "\n";
```

**Comment by scottchiefbaker** [comment]
If I modify the line that calculates max width to this:

```
my $max = List::Util::max(map {defined $$_[$i] ? length(colorstrip($$_[$i])) : 0} @$rows);
```

It calculates the correct width, **but** the `printf` line that uses `$format` collapses because the "length" of the string is larger than the length of the given parameter. In this case the second column has a max width of 8, but the `length` of that ANSI row is more than 8 so `printf` collapses it. I'm not sure of a good way to fix this without switching away from `printf`.

**Comment by scottchiefbaker** [comment]
I'm not the only with with ANSI / printf issues, but it doesn't seem like there is a good solution other than **not** using `printf`

https://stackoverflow.com/questions/5412761/using-colors-with-printf
https://stackoverflow.com/questions/3723429/how-to-stop-ansi-colour-codes-messing-up-printf-alignment
http://www.perlmonks.org/?node_id=1006953

**Comment by neilb** [comment]
Fixed in 1.00, now on CPAN.
